app/views/providers/open_banking_guidances/show.html.erb

```diff
@@ -12,7 +12,7 @@
     <p class="govuk-body"><%= t('.your_client_can') %></p>
     <p class="govuk-body"><%= t('.its_a_fast') %></p>
 
-    <details class="govuk-details" data-module="govuk-details">
+    <details class="govuk-details no-print" data-module="govuk-details">
```

Comment:
May I suggest using the [GOV.UK Design System display override](https://design-system.service.gov.uk/styles/layout/#override-how-elements-display) `govuk-!-display-none-print` instead of `no-print`

Reply:
Thanks! Have amended.